src/components/Chip/Chip.tsx

```diff
@@ -34,48 +36,47 @@ class Chip extends React.PureComponent<Props, any> {
       theme,
     } = this.props;
 
-    let className;
+    const className = classNames(
+      theme.Chip,
+      transparent && theme.transparent
+    );
 
-    if (this.props.markedForDelete) {
-      className = classNames(
-        theme.Chip,
-        theme.markedForDelete,
-        transparent && theme.transparent
-      );
-    } else {
-      className = classNames(
-          theme.Chip,
-          transparent && theme.transparent
-      );
-    }
+    const deleteInstruct = this.props.removable ? 'Press delete to remove this chip' : '';
 
     const chipContents = [(
       image
       ?
-      <img className={theme.Image} src={image.url} alt={image.alt} key="1" />
```

Comment:
@weegrom Do you know why the alt tag was removed?

Reply:
@harsimranb I'm just as confused as you as to why this line was removed.